### PR TITLE
Rename application

### PR DESCRIPTION
**Pull request by laisspportugal**
**User-Facing Changes**

Renames Foxbox to Lichtblick and removes some old links related to documentation of the tool.

**Description**
Since we have a new name, this PR aims to rename Foxbox to Lichtblick when it's visible to the user. We are also renaming places that mentions Foxglove Studio and it's documentation.

**Checklist**

- [x] The web version was tested and it is running ok
- [x] The desktop version was tested and it is running ok
- [x] I've updated/created the storybook file(s)
- [x] The release version was updated on package.json files

### CODE REVIEW

File: packages/studio-desktop/src/electronBuilderConfig.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ function makeElectronBuilderConfig(params) {
         },
         {
           ext: "foxe",
-          name: "Foxbox Extension",
+          name: "Lichtblicklick Extension",
```
Comment (Simsalabim1234) [inline packages/studio-desktop/src/electronBuilderConfig.js:51]: "Lichtblickklick" intended?

Comment (laisspportugal) [reply packages/studio-desktop/src/electronBuilderConfig.js:51]: No, thanks for the hint. This PR is an work in progress :)

Comment (laisspportugal) [reply packages/studio-desktop/src/electronBuilderConfig.js:51]: Fixed :)

File: packages/studio/README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
-# Foxbox Extension API
+# Lichtblick Extension API
 
-This package contains type definitions for writing [Foxbox](https://foxglove.dev/) extensions.
+This package contains type definitions for writing [Lichtblick](https://foxglove.dev/) extensions.
 
 See https://docs.foxglove.dev/docs/visualization/extensions/introduction
```
[the remark this answers is not in the document]
Comment (laisspportugal) [reply packages/studio/README.md]: removed

**Review by aneuwald-ctw — APPROVED** [review]

File: packages/hooks/package.json [hunk 3]
```diff
@@ -4,11 +4,11 @@
...
-    "url": "https://github.com/foxglove/studio.git"
+    "url": "https://github.com/bmw-software-engineering/lichtblick.git"
   },
   "author": {
-    "name": "Foxglove Technologies",
-    "email": "support@foxglove.dev"
+    "name": "Lichtblick",
+    "email": "lichtblick@bmwgroup.com"
   },
   "homepage": "https://foxglove.dev/",
```
Comment (Simsalabim1234) [inline packages/hooks/package.json:13]: Shouldn't this be also changed?

**Review by aneuwald-ctw — APPROVED** [review]

**Pull request merged**